java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaObjectOutputStream.java

```diff
@@ -118,7 +125,7 @@ public Thread newThread(final Runnable runnable) {
     /**
      * A running count of the total number of bytes written.
      */
-    private AtomicLong bytesWritten = new AtomicLong(0L);
+    private LongAdder bytesWritten = new LongAdder();
```

Comment:
I'm not sure of the reason for this change.  A LongAdder is for cases where being "maybe a little wrong" is okay under heavy contention.  But shouldn't the content-type header be exactly correct?  Do we expect extreme contention?